velox/functions/prestosql/aggregates/MinMaxByAggregates.cpp

```diff
@@ -107,42 +232,38 @@ class MinMaxByAggregate : public exec::Aggregate {
       const SelectivityVector& rows,
       const std::vector<VectorPtr>& args,
       MayUpdate mayUpdate) {
-    // decodedValue will contain the values of column X.
-    // decodedComparisonValue will contain the values of column Y which will be
-    // used to select the minimum or the maximum.
-    DecodedVector decodedValue(*args[0], rows);
-    DecodedVector decodedComparisonValue(*args[1], rows);
-
-    if (decodedValue.isConstantMapping() &&
-        decodedComparisonValue.isConstantMapping()) {
-      if (decodedComparisonValue.isNullAt(0)) {
-        return;
-      }
-      auto value = decodedValue.valueAt<T>(0);
-      auto comparisonValue = decodedComparisonValue.valueAt<U>(0);
-      auto nullValue = decodedValue.isNullAt(0);
-      rows.applyToSelected([&](vector_size_t i) {
-        updateValues(groups[i], value, comparisonValue, nullValue, mayUpdate);
-      });
-    } else if (
-        decodedValue.mayHaveNulls() || decodedComparisonValue.mayHaveNulls()) {
+    /// decodedValue will contain the values of column X.
```

Comment:
Use /// for method and class-level comments and // for in-the-body comments.

pre-existing, but would be nice to change "will contain" to "contains" and "which will be" -> "which is"